activities/ChatPrototype.activity/js/activity.js

```diff
@@ -45,7 +45,9 @@ define(["sugar-web/activity/activity","webL10n","sugar-web/graphics/palette","su
 		// Connect to network
         var presenceObject;
 		function shareActivity() {
+      			presencepalette.popDown();
```

Comment:
You have changed the indentation of this line compared to the file. Could you make sure that it is all in line?